assets/src/blocks/HubspotForm/HubspotFormFrontend.js

```diff
@@ -0,0 +1,70 @@
+import { useRef, useEffect, useState } from '@wordpress/element';
+import { useHubspotForm } from './hooks/useHubspotForm';
+import { useBackgroundImage } from './hooks/useBackgroundImage';
+
+const { __ } = wp.i18n;
+
+export const HubspotFormFrontend = ({
+  formTitle,
+  formText,
+  blockBackgroundImageUrl,
+  blockTitle,
+  blockText,
+  blockStyle,
+  ctaLink,
+  ctaText,
+  ctaNewTab,
+  hubspotShortcode,
+  hubspotThankyouMessage,
+  enableCustomHubspotThankyouMessage,
+}) => {
+  const hubspotFormRef = useRef(null);
+  const [ styleClass, setStyleClass ] = useState('');
+  const { submitted, submittedMessage } = useHubspotForm(hubspotShortcode, hubspotFormRef);
+  const [ backgroundImage ] = useBackgroundImage(blockBackgroundImageUrl);
+
+  useEffect(() => {
+    if(blockStyle) {
+      setStyleClass(blockStyle);
+    }
+  }, [ blockStyle ]);
+
+  return (
+    <section className={`hubspot-form block-wide ${styleClass}`} style={{...backgroundImage}}>
+      <div className='container'>
+        <div className='block-wrapper'>
+          <div className='block-wrapper-inner block-wrapper-text' style={{...backgroundImage}}>
```

Comment:
Is it necessary to use the backgroundImage twice (here and on the `section` element above)?

Reply:
Yes, since its layout is different between large and small screens.
Large screens -> The background image is set to the main wrapper (including block text and block form)
Smaller screens -> The background image is only set to the block text.

Have a look the [design](https://p4-designsystem.greenpeace.org/05f6e9516/p/213df0-hubspot-forms/b/99e047)